### PR TITLE
Feature/model drinking

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: src/TransformCore/PHE/HayApi/PersistenceBundle/Entity/Questions/Drinking.php [hunk 1]
```diff
@@ -15,5 +16,82 @@
...
+    public function getWeekdayDrinks()
+    {
+        return $this->weekdayDrinks;
+    }
+
+    /**
+     * @param Drinks $weekdayDrinks
+     * @return Drinking
+     */
+    public function setWeekdayDrinks($weekdayDrinks)
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Entity/Questions/Drinking.php]: Typehint?

**Comment by eddiejaoude** [comment]
Few TypeHint / Docbloc issues

File: src/TransformCore/PHE/HayApi/PersistenceBundle/Controller/DrinkingController.php [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+     *  default=0,
+     *  strict=true,
+     *  description="Number of shots of spirits per weekend."
+     * )
+     * @RequestParam(
+     *  name="weekend_wine_amount",
+     *  nullable=false,
+     *  default=0,
+     *  strict=true,
+     *  description="Number of glasses of wine per weekend."
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Controller/DrinkingController.php]: That is a lot of annotations :cry:

File: src/TransformCore/PHE/HayApi/PersistenceBundle/Repository/DrinkingRepository.php [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+     * @param ParamFetcher $paramFetcher
+     * @return Drinking
+     * @throws \Exception
+     */
+    public function updateDrinking(Drinking $drinking, ParamFetcher $paramFetcher)
+    {
+        $entityManager = $this->getEntityManager();
+        $entityManager->beginTransaction();
+
+        try {
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Repository/DrinkingRepository.php]: A big try/catch. Suggest, removing most of it out and only have the required part in there.

File: src/TransformCore/PHE/HayApi/PersistenceBundle/Controller/DrinkingController.php (hunk 2 above)
Comment (brettminnie) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Controller/DrinkingController.php]: Yeah, each parameter is an annotation...

File: src/TransformCore/PHE/HayApi/PersistenceBundle/Repository/DrinkingRepository.php (hunk 3 above)
Comment (brettminnie) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Repository/DrinkingRepository.php]: Ok, I'll try refactoring. Bear in mind I've used whitespace quite liberally here

**Pull request merged**